app/uk/gov/hmrc/customs/emailfrontend/forms/Validation.scala

```diff
@@ -21,24 +21,22 @@ import uk.gov.hmrc.customs.emailfrontend.Utils.stripWhiteSpaces
 
 object Validation {
 
-  final private val validEmailPattern = """^\s*([a-zA-Z0-9.!#$%&’'*+/=?^_`{|}~-]+)@([a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*)\s*$""".r
+  private val emailRegex = """^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$""".r
 
-  def validEmail: Constraint[String] =
+  def isValid(e: String): Boolean = e match{
+    case e if emailRegex.findFirstMatchIn(e).isDefined  => true
+    case _                                              => false
+  }
+
+  def isValidEmail: Constraint[String] =
     Constraint({
-      case e if stripWhiteSpaces(e).isEmpty =>
-        Invalid(ValidationError("customs.emailfrontend.errors.valid-email.empty"))
-      case e if e.length > 50 =>
-        Invalid(ValidationError("customs.emailfrontend.errors.valid-email.too-long"))
-      case e if !isValid(stripWhiteSpaces(e)) =>
-        Invalid(ValidationError("customs.emailfrontend.errors.valid-email.wrong-format"))
+      case null => Invalid(ValidationError("customs.emailfrontend.errors.valid-email.empty"))
```

Comment:
Better to use as below instead of using null as it is not considered as good practice
Ex - 
Option(a).isEmpty